scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/modules/Caster.kt

```diff
@@ -29,8 +30,13 @@ class Caster @Inject constructor(
     private fun waitForAnimationToFinish(timeout: Duration = 5.seconds) {
         val img = images[Images.BattleScreen]
 
-        // slow devices need this. do not remove.
-        locations.battle.screenCheckRegion.waitVanish(img, 2.seconds)
+        if (prefs.gameServer == GameServerEnum.Kr) {
+            // Skip Skill Animation. See #1319
+            locations.battle.skipAnimationClick.click()
+        } else {
+            // slow devices need this. do not remove.
+            locations.battle.screenCheckRegion.waitVanish(img, 2.seconds)
+        }
 
```

Comment:
I would move this out of the else, just to be sure.
```suggestion
        }
        
        // slow devices need this. do not remove.
        locations.battle.screenCheckRegion.waitVanish(img, 2.seconds)
```